css/_components.jsconfeu.css

```diff
@@ -0,0 +1,9 @@
+.c-jsconfeu {
```

Comment:
If we want to use namespaces, shall we add a link to the README, e.g. this one? http://csswizardry.com/2015/08/bemit-taking-the-bem-naming-convention-a-step-further/#namespaces
So we make our class structure transparent.